lib/Fissure.js

```diff
@@ -137,4 +137,6 @@ module.exports = class Fissure extends WorldstateObject {
   toString() {
     return `[${this.getETAString()}] ${this.tier} fissure at ${this.node} - ${this.enemy} ${this.missionType}`;
   }
-};
+}
+
+module.exports = Fissure;
```

Comment:
why?

Reply:
sorry, should've phrased it better.

why doesn't typescript resolve the type without this

Reply:
This fixes issue with typescript type resolution, moving the export away from the declaration of the class fixes the issue

Here is another example with other classes that typescript has problem to resolve due to the syntax 

`module.exports = class ZarimanCycle`

![image](https://user-images.githubusercontent.com/61201884/234147540-0f408210-3b33-4613-9be3-8c07384cb21a.png)

Reply:
yeah, i was already committing that one to your branch, lol. gg typescript not working right.